packages/ui/src/Dashboard/DetailView/ActionsDetails/BuySell/BuySell.tsx

```diff
@@ -52,27 +56,23 @@ class BuySell extends Component<Props, State> {
       const response = await apiCall('/v1/order/place', 'POST', payload);
       const error = getErrorMessage(response);
       if (error) {
-        console.log('Error');
-        console.log(error);
-        console.log(payload);
-        console.log(response);
-      } else {
-        console.log('Placed');
-        console.log(payload);
-        console.log(response);
+        this.props.showMessagePopup('error', response.errors.join(' , '));
+      }
+      else {
+        this.props.showMessagePopup('success', response.data.status);
```

Comment:
can we have message as `Order Placed` instead?
Since, status could also be 'PartiallyFilled'(without spaces) and it's not ideal to show it in the UI.